contrib/seeds/makeseeds.py

```diff
@@ -127,7 +138,11 @@ def filterbyasn(ips, max_per_asn, max_total):
     return result
 
 def main():
-    js = json.load(sys.stdin)
+    if len(sys.argv) > 1:
```

Comment:
This seems more complicated than the previous version. Does it have any advantage over `<file.json contrib/seeds/makeseeds.py`?

Reply:
You can still do it the old style if you prefer it. The new style was just more convenient while debugging this script as my IDE was not able to pass in the input data through stdin

Reply:
I'm fine w/it, but personally prefer to have _less code_, rather than more (as I know you generally prefer also).